examples/kni/main.rs

```diff
@@ -1,11 +1,21 @@
+use metrics_core::{Builder, Drain, Observe};
+use metrics_runtime::observers::YamlBuilder;
+use nb2::metrics;
 use nb2::settings::load_config;
 use nb2::{batch, Result, Runtime};
+use std::time::Duration;
 use tracing::{debug, Level};
 use tracing_subscriber::fmt;
 
+fn print_stats() {
+    let mut observer = YamlBuilder::new().build();
+    metrics::global().controller().observe(&mut observer);
+    println!("{}", observer.drain());
```

Comment:
Also related to the syn flood example, maybe it's the Yamlbuilder? but we can print things w/o the escape symbols? It's kinda lame.

Reply:
unfortunately it's quote inside a quoted string, can't avoid it. we can definitely make a different observer that's more console print friendly. the current built in ones are kinda messy each in their own way.

Reply:
yeah, i think i rather output json anyhow, tbh, as the default, but hopefully we can clean this up eventually.

Reply:
I tried json already, it's not pretty formatted, no line breaks. way harder to read. prometheus format has too many line breaks hard to tell the iterations apart. yaml unfortunately is the most readable one out of the built in 3 until we do a console observer ourselves.

Reply:
huh. ok i guess.